### PR TITLE
Create node.js.yml

### PR DESCRIPTION
**Pull request by Lilainita**

### CODE REVIEW

**Comment by NicolasB2** [comment]
Remember to update the test script by adding -t 5000 (step 4)

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - name: Use Node.js 16.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build --if-present
+    - name: Run tests
+      run: npm test
+
```
Comment (danielgalvis98) [inline .github/workflows/node.js.yml:21]: Last line should not have any spaces, or any character. It should be just an empty line generated by the line break, representing the EOF

Comment (Lilainita) [reply .github/workflows/node.js.yml:21]: So, what should I do? 
Delete the file and start all over again? or Can I edit this file?

**Comment by Lilainita** [comment]
> Remember to update the test script by adding -t 5000 (step 4)

Should I create this branch locally to push the commit to this PR? I'm a little bit lost...

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (danielgalvis98) [reply .github/workflows/node.js.yml:21]: I recommend you to pull this branch locally, continue working there and edit all the files locally. Then commit and push to this branch

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**